oci/oci_linux.go

```diff
@@ -0,0 +1,57 @@
+// +build linux
+
+package oci
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"syscall"
+
+	"github.com/containerd/cgroups"
+	"github.com/kubernetes-incubator/cri-o/utils"
+	rspec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sys/unix"
+)
+
+func (r *Runtime) createContainerPlatform(c *Container, cgroupParent string, pid int) error {
+	// Move conmon to specified cgroup
+	if r.cgroupManager == SystemdCgroupsManager {
+		logrus.Infof("Running conmon under slice %s and unitName %s", cgroupParent, createUnitName("crio-conmon", c.id))
+		if err := utils.RunUnderSystemdScope(pid, cgroupParent, createUnitName("crio-conmon", c.id)); err != nil {
+			logrus.Warnf("Failed to add conmon to systemd sandbox cgroup: %v", err)
+		}
+	} else {
+		control, err := cgroups.New(cgroups.V1, cgroups.StaticPath(filepath.Join(cgroupParent, "/crio-conmon-"+c.id)), &rspec.LinuxResources{})
+		if err != nil {
+			return fmt.Errorf("Failed to add conmon to cgroupfs sandbox cgroup: %v", err)
```

Comment:
Shouldn't errors start with lowercase strings?  For example, [here's Go's `lint` checking for lowercase errors][1].

[1]: https://github.com/golang/lint/blob/85993ffd0a6cd043291f3f63d45d656d97b165bd/lint.go#L1206-L1208

Reply:
> maybe so, but this is copy of the existing code. Was it failing lint before?

It [wasn't an error before][1], but I [like that you've turned it into one][2] here.

[1]: https://github.com/kubernetes-incubator/cri-o/pull/1407/files#diff-87558c1a90a59158e2c18a316e0c6693L239
[2]: https://github.com/kubernetes-incubator/cri-o/pull/1407#discussion_r179027985

Reply:
good catch, i'm switching this back to `logrus.Warnf()`